### DISCUSSION TITLE
Pool not showing up across accounts on multiple computers using same keys

### DISCUSSION THREAD

**Discussion by ZielReaver** [Chia-Network/chia-blockchain#7154] [Support]
I have three computers and all three using say account keys. Two of the computers when I updated to 1.1.75 and loaded the pool show up using the same name and launcher ID. For some reason the third computer did not bring up same name but instead created a new name and new launcher ID. The first two computers are now showing the original pool and the pool that created from third computer. But third computer still not showing original pool.

Please help me as I am wanting all three accounts to use the same pool ID. Only solution I have at this time is to abandons the original pool ID that I have been plotting to all day and start using the ID the third computer created as that is only one showing up across all three systems. Is there anyway I can get third system to register the original ID as I do not want to lose the almost 200 plots I have on it

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
